package.json

```diff
@@ -12,7 +12,7 @@
     "build:static": "next build && next export",
     "start": "next build && next start",
     "start:watch": "nodemon --exec \"npm run start\"",
-    "cname": "echo 'opencerts.io' > out/CNAME",
```

Comment:
Can remove. This was previously used for git pages....

Reply:
fixed.